components/survey_monkey/survey_monkey.app.mjs

```diff
@@ -101,27 +101,32 @@ export default {
         ...otherConfig,
       };
 
-      return axios($ || this, config);
+      return axios($, config);
     },
-    async _paginatedRequest(params) {
+    async _paginatedRequest(args = {}) {
       // https://api.surveymonkey.net/v3/docs?javascript#pagination
       // using default SurveyMonkey pagination - 50 per page
-      const amountPerPage = 50;
+      const amountPerPage = 1;
```

Comment:
Why did you change `50` to `1`?

Reply:
That was a mistake - I had changed it for testing and should not have commited that change. Fixed this, thanks!